validator/validator-main.protoascii

```diff
@@ -2870,6 +2870,7 @@ tags {
   spec_name: "FORM > DIV [submit-success]"
   mandatory_parent: "FORM"
   attrs: { name: "align" }
+  attrs: { name: "template" }
```

Comment:
Let's add new tests to `validator/testdata/feaure_tests/forms.html` and `validator/testdata/feaure_tests/forms.out` to test the validator change.